ibis/backends/flink/__init__.py

```diff
@@ -349,6 +350,12 @@ def create_table(
             dictionary of key-value pairs (key1=val1, key2=val2, ...).
         watermark
             Watermark strategy for the table, only applicable on sources.
+        primary_key
+            Column or comma-separated list of columns to be marked as PRIMARY.
```

Comment:
I *think* what we'd want here is a single column as a string, or multiple columns as a sequence of strings.

```python
con.create_table(..., primary_key="foo")
con.create_table(..., primary_key=["foo", "bar"])
```

Right now it looks like this always takes a string, which may happen to be comma delimited (e.g. `primary_key="foo,bar"`).

Reply:
Thanks, changed the function signature as you suggested.